chemprop/hyperopt_utils.py

```diff
@@ -116,29 +169,38 @@ def get_hyperopt_seed(seed: int, dir_path: str) -> int:
 def load_manual_trials(manual_trials_dirs: List[str], param_keys: List[str], hyperopt_args: HyperoptArgs) -> Trials:
     """
     Function for loading in manual training runs as trials for inclusion in hyperparameter search.
-    Trials must be consistent in all arguments with trials that would be generated in hyperparameter optimization.
+    Trials must be consistent with trials that would be generated in hyperparameter optimization.
+    Parameters that are part of the search space do not have to match, but all others do.
 
     :param manual_trials_dirs: A list of paths to save directories for the manual trials, as would include test_scores.csv and args.json.
     :param param_keys: A list of the parameters included in the hyperparameter optimization.
     :param hyperopt_args: The arguments for the hyperparameter optimization job.
     :return: A hyperopt trials object including all the loaded manual trials.
     """
-    matching_args = [ # manual trials must occupy the same space as the hyperparameter optimization search. This is a non-extensive list of arguments to check to see if they are consistent.
-        'number_of_molecules',
-        'aggregation',
-        'num_folds',
-        'ensemble_size',
-        'max_lr',
-        'init_lr',
-        'final_lr',
-        'activation',
-        'metric',
-        'bias',
-        'epochs',
-        'explicit_h',
-        'reaction',
-        'split_type',
-        'warmup_epochs',
+    # Non-extensive list of arguments that need to match between the manual trials and the search space.
+    matching_args = [ 
+        ('number_of_molecules', None),
+        ('aggregation', 'aggregation'),
+        ('num_folds', None),
+        ('ensemble_size', None),
+        ('max_lr', 'max_lr'),
+        ('init_lr', 'init_lr_ratio'),
+        ('final_lr', 'final_lr_ratio'),
+        ('activation', 'activation'),
+        ('metric', None),
+        ('bias', None),
+        ('epochs', None),
+        ('explicit_h', None),
+        ('adding_h', None),
+        ('reaction', None),
+        ('split_type', None),
+        ('warmup_epochs', 'warmup_epochs'),
+        ('split_sizes', None),
+        ('aggregation_norm', 'aggregation_norm'),
+        ('batch_size', 'batch_size'),
+        ('depth', 'depth'),
+        ('dropout', 'dropout'),
+        ('ffn_num_layers', 'ffn_num_layers'),
     ]
```

Comment:
How about adding ('dataset_type', None), ('multiclass_num_classes', None), ('features_generator', None), ('no_features_scaling', None), ('features_only', None) into this list. Not sure if it is necessary to extend this list to check more parameters when manual trails are given.

Reply:
These are good additions, I will add them